commands-v2/ramsete/constants.py

```diff
@@ -0,0 +1,61 @@
+from commands2 import RamseteCommand
+
+from wpilib import ADXRS450_Gyro
+
+from wpimath.kinematics import DifferentialDriveKinematics
+
+from math import pi
+
+"""
+A place for the constant values in the code that may be used in more than one place. 
+This offers a convenient resources to teams who need to make both quick and universal
+changes.
+"""
+
+# ID for the driver's joystick.
+kDriverControllerID = 0
+
+# The CAN IDs for the drivetrain motor controllers.
```

Comment:
CAN IDs? These are PWM, right? Is this an upstream bug?

Reply:
Oh that's correct. So used to writing CAN IDs, haha will fix.